spec/index.html

```diff
@@ -1278,6 +1282,11 @@ <h3>Escape Sequences</h3>
             <a href="#grammar-production-HEX"><code>HEX</code></a>
             ::= [0-9] | [A-F] | [a-f]</span></p>
         </blockquote>
+        <p>
+          Unicode <a data-cite="I18N-GLOSSARY#dfn-surrogate" class="lint-ignore">surrogate code points</a>,
+          the range <code class="codepoint">U+D800</code> to <code class="codepoint">U+DFFF</code>
+          are excluded.
```

Comment:
This prevents paired surrogates to be used (see #83). It sounds like a breaking change from Turtle 1.1 that allowed paired surrogates to be escaped.

Reply:
The change happened when "RDF string" came in where it uses Unicode scale values.

Given that surrogate pairs are translated by UTF-16 only, if a pair is introduced by escape sequences, I don't see how the 1.1 spec allows for a translation to be done; so they remain raw (and can't be written in UTF-8).

RDF 1.1 Turtle says "Unicode character" - which isn't a defined term (this may be due to improvements in Unicode).
A surrogate isn't what Unicode calls an "abstract character" (i.e. after decoding).

So, for me, this can be treated as errata on RDF 1.1 Turtle.

https://www.unicode.org/versions/Unicode16.0.0/core-spec/chapter-2/#G14527

>Surrogate
> Permanently reserved for UTF-16; restricted interchange
> Cannot be assigned to abstract character

Reply:
Indeed. Thank you for the clarification!

Reply:
OT: The Wikipedia disambiguation page for "Down the Rabbit Hole" refers ("see also") to the disambiguation page for "Rabbit Hole" ... which in turn refers to the disambiguation page for "Down the Rabbit Hole".  I hope someone did that on purpose.